autoplex/auto/phonons/jobs.py

```diff
@@ -327,7 +327,9 @@ def dft_phonopy_gen_data(
     jobs = []
     dft_phonons_output = {}
     dft_phonons_dir_output = []
-    supercell_matrix = reduce_supercell_size(structure, **supercell_settings)
+    supercell_matrix = supercell_settings.get(structure, {}).get("supercell_matrix")
```

Comment:
@JaGeo , I could make it that for each structure one could pass a supercell_matrix like:
```python
supercell_settings = {
    structure: {
        "supercell_matrix": [[0, 2, 0], [0, 0, 2], [2, 0, 0]]
    },
    "min_length": 12,  
    "max_length": 25  
}
```

Reply:
Oh, the Structure object is unhashable of course...

Reply:
Just use a list or use the mpid. And then use the number in the list/mpid to identify the supercell

Reply:
I switched to the mpids:
```python
supercell_settings = {
    mp_id: {
        "supercell_matrix": [[0, 2, 0], [0, 0, 2], [2, 0, 0]]
    },
    "min_length": 12,  
    "max_length": 25  
}
```
@JaGeo  what do you think about that?

Reply:
Would you like me to add or change something? I think this is the best option to make the user set each supercell_matrix individually if they want to have a different one.

Reply:
Yes, agreed.

Reply:
Ok great, then I will add a unit test and documentation 😃